core/events/events_selected.js

```diff
@@ -26,12 +27,11 @@ goog.require('Blockly.utils.object');
  *    element currently selected (deselect). Undefined for a blank event.
  * @param {string=} opt_workspaceId The workspace identifier for this event.
  *    Null if no element previously selected. Undefined for a blank event.
- * @extends {Blockly.Events.UiBase}
+ * @extends {UiBase}
  * @constructor
  */
-Blockly.Events.Selected = function(opt_oldElementId, opt_newElementId,
-    opt_workspaceId) {
-  Blockly.Events.Selected.superClass_.constructor.call(this, opt_workspaceId);
+const Selected = function(opt_oldElementId, opt_newElementId, opt_workspaceId) {
```

Comment:
We don't have to do it right now, but I think this falls under the category of classes that could use a rename. I think `SelectedEvent` makes a lot more sense.

Reply:
+1, that would definitely make it clearer. Added a note to #5073 so we don't forget to revisit this.